integration_tests/integration/probationPractitionerReferrals.spec.js

```diff
@@ -0,0 +1,73 @@
+import sentReferralFactory from '../../testutils/factories/sentReferral'
```

Comment:
There's actually a `monitor.spec` that has PP tests in it, sorry (I thought it was a bit nicer to have smaller chunks based on part of user journey, rather than the whole user journey) - would you mind adding this test to that instead?

Reply:
I'm wrong, this is part of the refer journey so it does make sense to have stuff in this file...